### ISSUE TITLE
Broken in DBAL 2.6.x-dev

### ISSUE THREAD

**Issue by jonny-no1** [VasekPurchart/Doctrine-Date-Time-Immutable-Types#8]
```
ConversionException in ConversionException.php line 95:
Could not convert PHP value of type 'DateTimeImmutable' to type 'datetime'. Expected one of the following types: null, DateTime
```

**Comment by jonny-no1** [comment]
https://github.com/VasekPurchart/Doctrine-Date-Time-Immutable-Types/issues/7 reports incompatibility with 2.6.x too.

**Comment by VasekPurchart** [comment]
@foaly-nr1 Hello, from what I see, this is the issue with the unreleased changes in DBAL master as well as described in #7, right? I presume you are setting this dependency in your project? (was wondering for a moment if maybe someone started including it when you mentioned Symfony 3.2) - what is the reason to use an unstable version?

Btw have you read trough #7 pls? Is there something you can add, a new idea perhaps?

As soon as 2.6 is released this issue will be easy to solve, but until then I don't see a good way.

**Comment by jonny-no1** [comment]
I got a few things wrong at first. Yes, I set the doctrine/orm 2.6.x dependency in my project as it's not in production any time soon and I want to use some of the new features. It worked fine, but broke with https://github.com/doctrine/doctrine2/commit/6af1d2843f003b60300bf45edd41bc22b6407f6b that caused doctrine/common to be upgraded from 2.6.2 to 2.7.1.

Haven't figured out why doctrine/common 2.7.1 breaks this bundle, but would be interesting. Maybe I'll get to the bottom if it.
